node-fs-writefile/cp.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+const note = process.argv[2];
+const copied = process.argv[3];
+
+fs.readFile(note, (err, data) => {
+  if (err) throw err;
+  fs.writeFile(copied, `${data}\n`, err => {
+    if (err) throw err;
+  });
+});
```

Comment:
When making a copy of a file, it's a bad idea to introduce changes in the copy (like adding a new character).